src/product/entities/product.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn } from 'typeorm'
+
+@Entity({
+    name: 'Products',
```

Comment:
products* that's the name of the table in database

Reply:
I thought you wanted starting with mayus